markdown/events/2021/bytesize-25-nf-core-ampliseq.md

```diff
@@ -27,3 +27,72 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[15:52](https://youtu.be/a0VOEeAvETs&t=952)
+Then also alpha and beta diversity indices are produced. Alpha diversity is a measure of each of the samples, how diverse the sequences in there are. You can see that for this test data set, it's a bit small, the samples originating from groundwater, have the lowest Shannon diversity index but the sediment samples here have the highest one. To all of these comparisons will be of course also statistical analyses provided in the output of the pipeline. Beta diversity plots show the difference in PCOA plots. What you can see here is that the replicates for the different kinds of sample sources (groundwater, river water, sediment and soil in our example data set) are nicely clustering and well separated from the others.
+
+[16:55](https://youtu.be/a0VOEeAvETs&t=1015)
+Additionally there will be quality control figures like alpha rarefaction curves, which show you if you have a sequence with sufficient output for your samples. As long as those curves will flatten out you know that you have sufficient sequencing data produced for that sample. All those results can be also found on the nf-core website in `/ampliseq/results` and you can have a look what is produced, how does it look like and there is also extensive documentation what each of those plots mean and what you can use it for. Those plots, of course, cannot show all kinds of complicated experimental setups, but they are very helpful for getting a first view on the data and are also a sort of quality control that what you are producing here makes sense. With this I want to finish and I want to thank you for your attention, my colleagues at QBiC , who also were involved in the development of the pipeline, nf-core, especially Daniel Lundin, helped a lot and the University of Tübingen Microbiology and Geomicrobiology who produced a lot of data, that I was then able to analyze and with this pipeline and produce it as good as possible.
+
+[18:33](https://youtu.be/a0VOEeAvETs&t=1113)
+(host) Thanks very much for that very clear and insightful talk Daniel. Are there any questions? If anyone in the audience has questions you can unmute yourself or even add questions to the chat and I can read them out. Okay I don't see anything coming up so that was extremely clear so I guess that's why people don't have questions but if anything crops up anywhere, you know where to reach Daniel. You can message him and continue the discussion on the bytesize channel and thanks again for joining. We will be back next week with another pipeline focused talk.
+
+[19:23](https://youtu.be/a0VOEeAvETs&t=1163)
+(question) We have one comment here okay there's a question. This person thanks you for the presentation and asks what type of license is associated with this kind of tool, for example the ampliseq pipeline.
+
+(answer) It is a CCPY license, like I think all the nf-core pipelines have, there are no restrictions whatsoever.
```

Comment:
I know I made a mistake here, it is not CCBY, that is only the overview figure, is it possible to note that this is wrong and its MIT license (default for nf-core pipelines I think)?

Reply:
I can add a comment